group_vars/osds.sample

```diff
@@ -131,7 +131,7 @@ dummy:
 #osd_containerized_deployment_with_kv: false
 #kv_type: etcd
 #kv_endpoint: 127.0.0.1
-#ceph_osd_docker_prepare_env: ""
+#ceph_osd_docker_prepare_env: "CEPH_DAEMON=OSD_CEPH_DISK_PREPARE"
```

Comment:
why do you have a different content now? 
both  `group_vars/all.docker.sample`  and `group_vars/osds.sample` should share the same content.